### PR TITLE
feat: install-minor-version

### LINKED ISSUE #8: In local templates, always install latest minor version of abell.
Currently, there are 2 local templates, default and minimal. After `npm install` or `yarn`, the version of the abell mentioned in their package.json is installed.

But, with this, we have to update this repository on every Abell release. So we need a way to always install the latest minor version (As per the [SemVer](https://semver.org))

This is a little tricky since there isn't a straightforward command for this. 

These are the steps we will have to follow:
1. Read the current version from package.json (Let's say It is `2.5.66`)
2. Extract the major version (`2` in our case)
3. Plus 1 the value
4. Run `npm install abell@"<3.0.0"` (which installs the latest version before 3.0.0)

If there is any better way, do comment here.

### PR DESCRIPTION
**Pull request by judicaelandria**
Fixes #8

### CODE REVIEW

File: lib/actions.js [hunk 1]
```diff
@@ -1,109 +1,152 @@
...
-  console.log(`\n${colors.green('>')} Installing Dependencies 📚\n\n`);
-  
-  const isWindows = /^win/.test(process.platform);
-  let installerCommand = [isWindows ? 'npm.cmd' : 'npm', ['install']];
-  if (installer === 'yarn') {
-    installerCommand = [isWindows ? 'yarn.cmd' : 'yarn'];
+  console.log(`\n${colors.green(">")} Installing Dependencies 📚\n\n`);
+
+  // checking if the template is local
+  if (template === "default" || template === "minimal") {
```
Comment (saurabhdaware) [inline lib/actions.js:89]: This condition checks if the template is local or not. the local template can have template value `default` or `minimal`.

File: lib/actions.js [hunk 2]
```diff
@@ -1,109 +1,152 @@
...
+    let isLocalTemplate = "local";
+
+    // if local, ignoring the command
+    if (!isLocalTemplate === "local") {
```
Comment (saurabhdaware) [inline lib/actions.js:93]: So we won't really need this condition since the check for local is already done above

Comment (judicaelandria) [reply lib/actions.js:93]: yeah, so should move the else condition then

**Review by saurabhdaware — APPROVED** [review]
Perfect! Thank you so much @judicaelandria :tada:

**Pull request merged**